.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm start & sleep 5 && npm test
+    - run: npm test
```

Comment:
<img width="367" alt="image" src="https://user-images.githubusercontent.com/38065883/170825648-84977dab-d176-444a-af2b-02b71212d7e1.png">
- align items vertically

Reply:
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/38065883/170825680-d24633c3-9b86-44f0-ab43-5035646f658a.png">
- images are too large on this screen